README.md

```diff
@@ -1,438 +1,20 @@
-# Specifikace předávání událostí
+# Neltia documentation
+![Logo](/images/netlia_280x60_black.png)
 
-Dokument popisuje způsob předávání událostí ze systému Netlia do aplikace partnera.
+### English
+* [netlia.com](https://netlia.com/en/senzory-netlia/) - datasheets and further informations
+* [Event forwarding](EventForwarding.md) - describes how Netlia system forwards events to the partner's application
+* Device communication - describes the behaviour of the devices and a description of the communication protocol *comming soon
 
-**Způsob komunikace je mezi Netlia a partnerem dohodnut a na straně Netlia nastaven.**
+The table of content in Github documents can be displayed as shown below.
 
-## Obsah
 
-Obsah můžete zobrazit následujícím způsobem:
+### Česky
+* [netlia.com](https://netlia.com/cs/senzory-netlia/) - datasheety a další informace
+* [Předávání událostí](EventForwarding_cz.md) - popisuje způsob předávání událostí ze systému Netlia do aplikace partnera
+* [Komunikace zařízení](deviceComunicationDocumentation.MD) - popisuje chování jednotlivých zařízení a popis společného komunikačního protokolu
 
-![Obsah](/images/table-of-contents.webp)
+Obsah dokumentů na GitHubu je možné zobrazit způsobem znázorněným níže.
```

Comment:
Tohle bych mozna dal az do tech jednotlivých dokumentaci.

Stejne to v kazde z nich musi byt protoze v nekterych pripadech budeme posilat primo odkaz na ten soubor a sem se nikdo koukat nebude.